modules/gcp_bigquery_masked_dataset/outputs.tf

```diff
@@ -18,7 +18,12 @@ output "masked_dataset_self_link" {
   value       = google_bigquery_dataset.google_bigquery_masked_dataset.self_link
 }
 
+output "masked_dataset_customer_managed_key_id" {
+  description = "The reference ID of the created customer managed key"
+  value       = google_kms_crypto_key.main_dataset_kms_crypto_key.id
```

Comment:
Value should be from Terraform resource with name masked_dataset_kms_crypto_key.

Reply:
ah good catch thanks!

Reply:
[fixed](https://github.com/honestbank/terraform-gcp-bigquery/pull/27/commits/1c915a377576566abf35d7785896f3546ade0b8c)